.github/workflows/trigger-ci-on-branch.yml

```diff
@@ -11,9 +11,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Set env
+      run: echo "CIRCLE_BRANCH=$(git symbolic-ref -q --short HEAD)" >> $GITHUB_ENV
```

Comment:
I don't know if it makes any difference, but the official repo make the use a bit different https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

both for how to set and how to use it.
like this:
```
steps:
  - name: Set the value
    id: step_one
    run: |
      echo "action_state=yellow" >> $GITHUB_ENV
  - name: Use the value
    id: step_two
    run: |
      echo "${{ env.action_state }}" # This will output 'yellow'
```

Reply:
It works either way
`${CIRCLE_BRANCH}` === `${{ env.CIRCLE_BRANCH }}`

I prefer the current version

Reply:
The create env is also different, it as both name and value
`echo "action_state=yellow" >> $GITHUB_ENV`

And it uses `|` at run, but that I don't know if makes a difference.

If you are sure it works, I can approved it, I am just not familiarized with gh actions.

Reply:
I just read it again, and it should work, I missed one thing.